drivers/bus/pci/pci.c

```diff
@@ -29,6 +30,8 @@ static NTSTATUS NTAPI PciPnpControl(IN PDEVICE_OBJECT DeviceObject, IN PIRP Irp)
 /*** PUBLIC ******************************************************************/
 
 PPCI_DRIVER_EXTENSION DriverExtension = NULL;
+BOOLEAN HasDebuggingDevice = FALSE;
+PCI_TYPE1_CFG_CYCLE_BITS PciDebuggingDevice[2] = {{{{0}}}};
```

Comment:
Why 2 entries "only"?
(also regarding the initialization, isn't `{{0}}` -- with less brackets -- enough?)

Reply:
> Why 2 entries "only"?

I keep it in sync with the HAL.

Brackets are necessary for building using GCC. Maybe it's worth considering a change of the struct layout.

Reply:
So, with 2 entries, does this mean that this can describe two debugging devices available at the same time?

Reply:
Yes, the second device can also be used in a KD library.